metadata.json

```diff
@@ -37,6 +37,10 @@
       "name": "puppetlabs/concat",
```

Comment:
Can go ahead and bump `puppetlabs/stdlib` to `< 7.0.0` in here.

Reply:
Unrelated change, but you're the owner.

Reply:
@pillarsdotnet Yeah, didn't know if you were going to force-push again :laughing: